pages/docs/quickstart.mdx

```diff
@@ -122,6 +122,8 @@ You can skip `optional extended message`
 | 3rd dropdown | Your repository (source)             | `your-username/LinkFree`     |
 | 4th dropdown | Your repository branch (source)      | `your-username-patch-1`      |
 
+If the Base repository (destination) from 1st dropdown is set to `your-username/LinkFree` change it to `EddieHubCommunity/LinkFree`
+
 ![compare across forls](https://user-images.githubusercontent.com/82668196/208292189-513b4213-a3a7-4a45-a834-ac6ce6b9217a.png)
```

Comment:
While you're at it, perhaps you could fix this typo. What do you think?

```suggestion
![compare across forks](https://user-images.githubusercontent.com/82668196/208292189-513b4213-a3a7-4a45-a834-ac6ce6b9217a.png)
```

Reply:
Oh, nice thanks for this!!!